lib/samples/show_service_area/show_service_area_sample.dart

```diff
@@ -80,84 +134,123 @@ class _ShowServiceAreaSampleState extends State<ShowServiceAreaSample> {
   }
 
   Future<void> onMapViewReady() async {
```

Comment:
I think we usually declare this `void onMapViewReady() async`. I think we only need `Future<void>` if somebody upstream is doing an `await` on this method, but we know this is intended only for the `onMapViewReady:` callback.